src/utils/DataRetrievalService.js

```diff
@@ -0,0 +1,73 @@
+//Retrieval function for mock files;
+export const DataRetrievalService = {
```

Comment:
Correct me if i'm wrong, you are planning on changing your API calls to something like what i showed you previously?

Reply:
I tried and spectacularly failed haha. I nope'd the heck out of that and went full API integration instead of trying to include the mock data and using a .env file to switch.